Objects/object.c

```diff
@@ -186,20 +210,10 @@ init_refchain(PyInterpreterState *interp)
 void
 _Py_AddToAllObjects(PyObject *op, int force)
 {
-#ifdef  Py_DEBUG
-    if (!force) {
-        /* If it's initialized memory, op must be in or out of
-         * the list unambiguously.
```

Comment:
Is `force` option needed at all for now?
Anyway, please update the comment above the function too.

Reply:
I don't understand well that flag. I would prefer to just remove it. It's not a big deal to add an object twice to the hash table, the second call does nothing. It's not like a linked list where the risk is to add the same object twice in the list.

Reply:
I removed the force parameter.